cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/parameters/ParameterUtils.java

```diff
@@ -352,11 +360,11 @@ public static boolean wantJSON(HttpServletRequest request) {
   }
 
   static boolean allowCapacityEstimation(HttpServletRequest request) {
-    return getBooleanParam(request, ALLOW_CAPACITY_ESTIMATION_PARAM, true);
+    return getBooleanExcludeGiven(request, ALLOW_CAPACITY_ESTIMATION_PARAM, REBALANCE_DISK_MODE_PARAM);
```

Comment:
Are we changing the default value of `ALLOW_CAPACITY_ESTIMATION_PARAM` from `true` to `false`?

* This is a behavior change -- and a critical behavior change we should definitely avoid.
Please refrain from such behavior changes in large patches -- i.e. create their own patch for review.

Reply:
It is not intentional... here I originally want to ensure that if `rebalance_disk=true`, `allow_capacity_estimation` should not be false. Now looking at this code I feel this might be too restrict, will revert the change here.